src/installer.js

```diff
@@ -44,6 +44,14 @@ var spawn = function (options, command, args, callback) {
   })
 
   spawnedProcess.on('error', function (err) {
+    if (err.name === 'ENOENT') {
+      var installer = process.platform === 'darwin' ? 'brew' : 'apt-get'
+
+      if (err.message.indexOf('fakeroot') > -1) {
```

Comment:
you could check `err.syscall` for `spawn fakeroot` or `spawn dpkg`, I do something similar in Electron Packager: https://github.com/electron-userland/electron-packager/blob/6f5f2934ca19a384991cc44b6c83e35322cbcd9d/win32.js#L29